drivers/pds/api/backupCredential.go

```diff
@@ -91,17 +92,20 @@ func (backupCredential *BackupCredential) CreateS3BackupCredential(tenantID stri
 		Name:        &name,
 	}
 	ctx, err := pdsutils.GetContext()
+	log.Infof("Ctx: %v", ctx)
 	if err != nil {
 		log.Errorf("Error in getting context for api call: %v\n", err)
 		return nil, err
 	}
 	backupModel, res, err := backupClient.ApiTenantsIdBackupCredentialsPost(ctx, tenantID).Body(createRequest).Execute()
+
+	if err != nil {
+		return nil, err
```

Comment:
log error similar to `fmt.Errorf("response code is not 200(OK) when called `ApiTenantsIdBackupCredentialsPost. Full response code %v", res)` it will be easy to debug

Reply:
done